chore(docker): address docker offline comments (#17377)
microsoft/playwright: docs/src/docker.md

    npx playwright docker start
    ```

1. Run tests inside Docker container using the `PLAYWRIGHT_DOCKER` environment variable:

    ```bash js

>     PLAYWRIGHT_DOCKER=1 npx playwright test

Reviewer: Drop this snippet.

    ```

   You can set this environment variable as a part of your config:

    ```ts
    // playwright.config.ts
    import type { PlaywrightTestConfig } from '@playwright/test';

    process.env.PLAYWRIGHT_DOCKER = '1';

    const config: PlaywrightTestConfig = {
      /* ... configuration ... */
    };
    export default config;
    ```

>    NOTE: this command will detect running Docker container, and auto-launch it if needed.

Reviewer: ```suggestion
   NOTE: Playwright will automatically detect a running Docker container or start it if needed.
```
